projects/valtimo/components/src/lib/components/select/select.component.ts

```diff
@@ -132,7 +132,7 @@ export class SelectComponent implements OnInit, OnChanges, OnDestroy {
 
     if (Array.isArray(selectedValue)) {
       this.setSelectedValue(selectedValue.map(value => value.id));
-    } else if (id && typeof id === 'string') {
+    } else if (selectedValue && typeof id === 'string') {
```

Comment:
this feels like it will lead to problems elsewhere in application, have you checked?